application/core/LSHttpRequest.php

```diff
@@ -36,7 +36,7 @@ class LSHttpRequest extends CHttpRequest
 {
 
     private $_pathInfo;
-    
+
```

Comment:
Please remove all white-space-only changes, they kinda pollute the PR and makes it harder to read.